src/main/resources/DDL_CreateGpfdTablesMojfin.sql

```diff
@@ -55,5 +55,9 @@ Insert into GPFD.CSV_TO_SQL_MAPPING_TABLE (ID,REPORT_NAME,SQL_QUERY,BASE_URL,REP
 Insert into GPFD.CSV_TO_SQL_MAPPING_TABLE (ID,REPORT_NAME,SQL_QUERY,BASE_URL,REPORT_OWNER,REPORT_CREATOR,REPORT_DESCRIPTION,EXCEL_REPORT,EXCEL_SHEET_NUM,CSV_NAME,OWNER_EMAIL) values (7,'CCMS AP Debtors','SELECT * FROM ANY_REPORT.V_CIS_TO_CCMS_INVOICE_CCMS_AP_DEBTS','https://justiceuk.sharepoint.com/sites/FinanceSysReference/SitePages/GPFD-Test-Site-page.aspx','William Moran','William Unwin','Details of invoices transferred from CIS to CCMS by Legal Aid Scheme','CCMS_AP_DEBTORS',7,'MAIN','William.Moran@Justice.gov.uk');
+Insert into GPFD.CSV_TO_SQL_MAPPING_TABLE (ID,REPORT_NAME,SQL_QUERY,BASE_URL,REPORT_OWNER,REPORT_CREATOR,REPORT_DESCRIPTION,EXCEL_REPORT,EXCEL_SHEET_NUM,CSV_NAME,OWNER_EMAIL) values (8,'CCMS_and_CIS_Bank_Account_Report_w_Category_Code_YTD','SELECT * FROM ANY_REPORT.V_BANK_YTD','https://justiceuk.sharepoint.com/sites/FinanceSysReference/SitePages/GPFD-Test-Site-page.aspx','William Moran','William Unwin','Summary of all payments made by CCMS/CIS and all cash receipts applied to debt YTD','CCMS_and_CIS_Bank_Account_Report_w_Category_Code_YTD',8,'MAIN','William.Moran@Justice.gov.uk');
+Insert into GPFD.CSV_TO_SQL_MAPPING_TABLE (ID,REPORT_NAME,SQL_QUERY,BASE_URL,REPORT_OWNER,REPORT_CREATOR,REPORT_DESCRIPTION,EXCEL_REPORT,EXCEL_SHEET_NUM,CSV_NAME,OWNER_EMAIL) values (9,'CCMS General Ledger (tiny columns multiple periods)','SELECT * FROM ANY_REPORT.V_CCMS_GL_EXTRACTOR_YTD','https://justiceuk.sharepoint.com/sites/FinanceSysReference/SitePages/GPFD-Test-Site-page.aspx','William Moran','William Unwin','CCMS General ledger extractor (tiny columns multiple periods)','CCMS_GEN_LEDGER_TINYCOL_MULPERIODS',9,'MAIN','William.Moran@Justice.gov.uk');
+Insert into GPFD.CSV_TO_SQL_MAPPING_TABLE (ID,REPORT_NAME,SQL_QUERY,BASE_URL,REPORT_OWNER,REPORT_CREATOR,REPORT_DESCRIPTION,EXCEL_REPORT,EXCEL_SHEET_NUM,CSV_NAME,OWNER_EMAIL) values (10,'CCMS General Ledger (tiny columns multiple periods manual)','SELECT * FROM ANY_REPORT.V_CCMS_GL_EXTRACTOR_MANUAL_YTD','https://justiceuk.sharepoint.com/sites/FinanceSysReference/SitePages/GPFD-Test-Site-page.aspx','William Moran','William Unwin','CCMS General ledger extractor (tiny columns multiple periods manual)','CCMS_GEN_LEDGER_TINYCOL_MULPERIODS_MANUAL',10,'MAIN','William.Moran@Justice.gov.uk');
+Insert into GPFD.CSV_TO_SQL_MAPPING_TABLE (ID,REPORT_NAME,SQL_QUERY,BASE_URL,REPORT_OWNER,REPORT_CREATOR,REPORT_DESCRIPTION,EXCEL_REPORT,EXCEL_SHEET_NUM,CSV_NAME,OWNER_EMAIL) values (11,'CCMS Authorised expenditure (Contracting live system)','SELECT * FROM ANY_REPORT.V_CCMS_AUTHORISED_INVS_CONTRACTING','https://justiceuk.sharepoint.com/sites/FinanceSysReference/SitePages/GPFD-Test-Site-page.aspx','William Moran','William Unwin','CCMS Authorised expenditure (Contracting live system)','CCMS_AUTH_EXPEND_CONTRACTING',11,'MAIN','William.Moran@Justice.gov.uk');
```

Comment:
Are we able to condense down to a single INSERT INTO with multiple values? Same as data.sql file?

Reply:
not with this version of ORACLE sadly.